web/src/contexts/configContext.ts

```diff
@@ -1,3 +1,5 @@
+import * as cannabisLicenseTab1 from "@businessnjgovnavigator/content/fieldConfig/cannabis-license-tab1.json";
+import * as cannabisLicenseTab2 from "@businessnjgovnavigator/content/fieldConfig/cannabis-license-tab2.json";
```

Comment:
small nitpick but I think these should be capitalized first letter to match the others